.gitignore

```diff
@@ -160,6 +160,13 @@ ajax/libs/Hyphenator/ersion_*
 ajax/libs/gemma/0.1.0/
 ajax/libs/gemma/0.4.0/
 ajax/libs/origamijs/0.0.2/
+ajax/libs/mammoth/0.1.0/
+ajax/libs/mammoth/0.1.1/
+ajax/libs/mammoth/0.1.2/
+ajax/libs/mammoth/0.1.3/
+ajax/libs/mammoth/0.1.4/
+ajax/libs/mammoth/0.1.5/
+ajax/libs/mammoth/0.1.6/
```

Comment:
Looks like we don't need theses mammoth since their files won't be grabbed by the current rule of auto-update.